public_html/wp-content/mu-plugins/blocks/assets/src/speakers/speakers-select.js

```diff
@@ -192,6 +203,8 @@ const optionsSelect = ( select, props ) => {
 	let options = [];
 
 	if ( ! mode || 'specific_terms' === mode ) {
+		// should there be some kind of UI to let the user know they can only select groups at the start, and can't mix speakers/groups?
+
```

Comment:
To clarify, you _can_ switch from speakers to groups, or vice versa, if you remove all of the selected items first. Good question, though. I think this another thing that would be good to discuss with the designer.